src/Workspaces/Core/Desktop/Workspace/FileTextLoader.cs

```diff
@@ -11,11 +11,17 @@
 
 namespace Microsoft.CodeAnalysis
 {
+    [System.Diagnostics.DebuggerDisplay("{GetDebuggerDisplay()}")]
```

Comment:
Use `DebuggerDisplay` without qualification and add a using for `System.Diagnostics`.